remotedata/src/main/java/com/mercari/remotedata/RemoteData.kt

```diff
@@ -89,7 +100,7 @@ internal inline fun <V : Any, E : Exception, U : Any, EE : Exception> RemoteData
 ): RemoteData<U, EE> =
         when (this) {
             RemoteData.NotAsked -> RemoteData.NotAsked
-            is RemoteData.Loading -> RemoteData.Loading()
+            is RemoteData.Loading -> RemoteData.Loading(progress)
```

Comment:
I am thinking about just use `this` what do you think about that?

Reply:
Hmmm, the inner type changes through the transform in this case, but we can do that in the `flatMapError` case.